### PR TITLE
Clean up pending withdraws in storage automatically in home and accou…

### PR DESCRIPTION
**Pull request by AlbertoElias**
…nt details

### CODE REVIEW

File: src/store/global/global.thunks.js [hunk 1]
```diff
@@ -83,7 +83,15 @@ function fetchFiatExchangeRates (symbols) {
   return (dispatch) => {
     dispatch(globalActions.loadFiatExchangeRates())
 
-    return fiatExchangeRatesApi.getFiatExchangeRates(symbols)
+    return Promise.resolve({
```
Comment (mmilenkovic) [inline src/store/global/global.thunks.js:86]: Wanna add a `TODO` here, so we don't forget to fix it?

File: src/store/global/global.thunks.js [hunk 2]
```diff
@@ -240,6 +247,37 @@ function checkPendingDelayedWithdraw (exitId) {
...
+      return CoordinatorAPI.getExit(pendingWithdraw.batchNum, pendingWithdraw.accountIndex)
+        .then((exitTx) => {
+          if (exitTx.instantWithdraw || exitTx.delayedWithdraw) {
+            dispatch(removePendingWithdraw(pendingWithdraw.id))
+          }
+        })
+        .catch(() => {})
+    }))
+      .then(() => dispatch(globalActions.checkPendingWithdrawalsSuccess()))
+      .catch(() => dispatch(globalActions.checkPendingWithdrawalsSuccess()))
```
Comment (mmilenkovic) [inline src/store/global/global.thunks.js:277]: Why success is returned in both cases?

File: src/store/global/global.thunks.js (hunk 1 above)
Comment (AlbertoElias) [reply src/store/global/global.thunks.js:86]: I'll create an issue

File: src/store/global/global.thunks.js (hunk 2 above)
Comment (AlbertoElias) [reply src/store/global/global.thunks.js:277]: It's fine if it fails, this is just an automated check to clean up pending withdrawals. I copied the structure from the funcion `checkPendingDeposits` that Elias did

**Review by mmilenkovic — APPROVED** [review]

**Pull request merged**